packages/story-utils/src/wait.ts

```diff
@@ -0,0 +1,9 @@
+import { act } from '@testing-library/react';
+
+export default async function wait(ms = 0): Promise<void> {
```

Comment:
This is implemented in `Along src/test/helpers.js`. Once this patch is merged, we can import it from here instead.